force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -293,22 +282,40 @@ public with sharing class ClassesObjectsMethods {
      * where you need to manipulate and manage person information.
      */
     public class Person {
-        // Declare private two instance variables - (TEXT) name and (Number) age
-        
-        // Constructor to initialize the name and age
-    
-        // Method introduceYourself that returns the details of the person        
-        // Return a string in the format "Hello, my name is <name> and I am <age> years old."
-        // For example, introduceYourself() should return a string like "Hello, my name is John Doe and I am 28 years old."
-
-        // Getter method that returns the name of the person
-
-        // Getter method that returns the age of the person
+        private String privateName;
+        private Integer privateAge;
+        private String introduceYourself;
```

Comment:
Definitely don't need to name these variables `privateName`. You can just call them `name`. We know they are private because we see it before their declaration.